src/icons/index.js

```diff
@@ -1,5 +1,7 @@
 const icons = {
   info: require('./Info').default,
+  uncheckedSquare: require('./UncheckedSquare').default,
```

Comment:
The reason I kept `uncheckedSquare` and `checkedSquare` is because we can have checkbox icon with round/circle in shape. The other day design team was presenting filters which had checkbox of circular type.